tests/unit_tests/test_server_client.py

```diff
@@ -661,6 +712,8 @@ async def test_astream_log_diff_no_effect(
                     "id": uuid,
                     "logs": {},
                     "streamed_output": [],
+                    "type": "chain",
```

Comment:
these are due to changes in behavior of astream_log in newer langchain core that added new fields